ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -677,3 +677,30 @@ def verify_storage_clients_page(self):
             page_name="client_onboarding_token_page",
         )
         return storage_client_obj
+
+    def odf_storagesystems_consumption_trend(self):
```

Comment:
pls put this method into BlockAndFile class

Reply:
This has dependency with `odf_console_plugin_check` from validation_ui and this function is being used in other two functions in the same file. PageNavigator() not found in 'block and file' which we may need to import it again. I think its better to keep it in the validation_ui only. wdyt?

Reply:
BlockAndFile inherits from PageNavigator via StorageSystemDetails, StorageSystemTab, etc. 
The Idea is that every page on OpenShift management-console should consist functions and locators visible on the browser when we do navigation there. 
So PageNavigator is a parent class for mostly every web page / page-object class .
Same for some other elements, search bars, lists, etc.

Reply:
done